tests/test_p2p_daemon.py

```diff
@@ -97,6 +100,27 @@ def handle_add(args):
     return result
 
 
+def handle_square_torch(x):
+    tensor = runtime_pb2.Tensor()
+    tensor.ParseFromString(x)
+    tensor = deserialize_torch_tensor(tensor)
```

Comment:
It would be convenient to create functions that deserialize the tensor from the raw bytes (`x` here) and serialize it from the raw bytes. You may put them to `hivemind/utils/compression.py` as well.

Reply:
Maybe do it apart? It doesn't affect behavior

Reply:
I'm okay with doing it in a later PR :)